### PR TITLE
Frontend fixes

### PR DESCRIPTION
**Pull request by ewanmay**

### CODE REVIEW

File: src/components/Component.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .center-button {
   position: absolute;
-  left: 48%;
+  left: calc(50% - 90px);
```
Comment (atatmaja) [inline src/components/Component.css:3]: why this change? did you test with different window sizes?

File: src/components/charts/Charts.tsx [hunk 2]
```diff
@@ -60,35 +74,34 @@ const renderCustomizedLabel = (props: any) => {
             </div>
             <Dropdown
               placeholder='Select Independent Variable'
-              defaultValue={yAxisOptions[0].text}
-              fluid
-              selection
-              item={true}
+              defaultValue={'1'}
+              fluid selection
+              // item={true}
```
Comment (atatmaja) [inline src/components/charts/Charts.tsx:79]: let's remove this comment if we're sure it won't be used

File: src/metaAnalysis.ts [hunk 3]
```diff
@@ -2,6 +2,11 @@ import { AirtableRecord } from "./types"
 
 const Z_SCORE: number = 1.96;
 
+export enum aggregationFactor {
```
Comment (atatmaja) [inline src/metaAnalysis.ts:5]: might be more consistent to put this in types.ts. not sure that's necessary tho, up to you

File: src/components/Component.css (hunk 1 above)
Comment (ewanmay) [reply src/components/Component.css:3]: essentially the button width is set to 180px; so it will always be centered with this calc
